lightning/src/offers/parse.rs

```diff
@@ -116,6 +116,8 @@ impl<T: SeekReadable> TryFrom<Vec<u8>> for ParsedMessage<T> {
 }
 
 /// Error when parsing a bech32 encoded message using [`str::parse`].
+///
+/// This is not exported to bindings users as its name conflicts with the BOLT 11 ParseError type.
```

Comment:
Does not exporting this affect using `FromStr` and `TryFrom` implementations for `Offer`?